pandas/tests/test_tseries.py

```diff
@@ -316,6 +316,29 @@ def test_maybe_booleans_to_slice():
     assert(result == slice(0, 0))
 
 
+def test_datetimelike_tostring_astype():
+    import pandas as pd
+    result = pd.date_range('2012-01-01', periods=4).astype(str)
+    expected = pd.Index(['2012-01-01', '2012-01-02', '2012-01-03', '2012-01-04'], dtype=object)
+    tm.assert_index_equal(result, expected)
+
+
+def test_datetimelike_tostring_astype_freqH():
+    import pandas as pd
```

Comment:
you don't need any of the imports, though you may need ``_skip_if_no_pytz``. below